### DISCUSSION TITLE
How to specify sessionVariables when connent to mysql compatible storage

### DISCUSSION THREAD

**Discussion by wfxxh** [apache/superset#16222] [Q&A / Help]
I use tidb as the analysis data storage and I want to specify the "sessionVariables=tidb_isolation_read_engines=tiflash" when I create the SQLAlchemy URI , but it is going to be error . How to config it?
![image](https://user-images.githubusercontent.com/22764286/128962053-7298bfd0-4609-4dc3-bda9-befaad5397b8.png)
![image](https://user-images.githubusercontent.com/22764286/128962181-3afa901f-b60d-4d76-ba03-ed643cc1bab7.png)

**Comment by sujanpgowda** [comment]
@wfxxh Same here, Were you able to use TiDB with superset? cc: @willbarrett

**Reply by wfxxh** [reply]
Yes，we use the tiflash engine
